zipline/finance/slippage.py

```diff
@@ -162,8 +239,17 @@ def asdict(self):
         return self.__dict__
 
 
-class VolumeShareSlippage(SlippageModel):
-    """Model slippage as a function of the volume of shares traded.
+class EquitySlippageModel(SlippageModel):
+    pass
+
+
+class FutureSlippageModel(SlippageModel):
+    pass
+
+
+class VolumeSlippage(object):
```

Comment:
d'oh, I'll change back. I think `VolumeShareSlippage` can just be used as is for both Equities and Futures